### PR TITLE
Patch v2.3.2

### PR DESCRIPTION
**Pull request by lionkor**
Several smaller patches, likely for upcoming public release, or next public release.

Changes such as Ctrl+C handling on windows (necessary for #32's `onShutdown` to work properly), better http status codes for cloudflare errors (will be easier to work with in sentry).

### CODE REVIEW

**Comment by lionkor** [comment]
See [Changelog.md](https://github.com/BeamMP/BeamMP-Server/blob/patch-v2.3.2/Changelog.md) for a full list of changes in this patch

**Comment by lionkor** [comment]
Not sure if this has enough features to be a 2.4.0 (?)

File: src/TConfig.cpp [hunk 1]
```diff
@@ -32,6 +33,13 @@ TConfig::TConfig() {
     }
 }
 
+void WriteSendErrors(const std::string& name) {
+    std::ofstream CfgFile { name, std::ios::out | std::ios::app };
+    CfgFile << "# If SendErrors is `true`, the server will send helpful info about crashes and other issues back to the BeamMP developers. This info may include your config, who is on your server at the time of the error, and similar data. You can opt-out of this system by setting this to `false`."
```
Comment (Starystars67) [inline src/TConfig.cpp]: We need to add an option to hide this text as well. Just for those who are picky :P

Comment (lionkor) [reply src/TConfig.cpp]: Yeah, on it right now - that's why its still a draft PR. Anything else I missed that we wanted to put?

**Comment by lionkor** [comment]
Needs some manual testing by others, but should be good to merge

**Review by Anonymous-275 — APPROVED** [review]

**Review by Starystars67 — APPROVED** [review]

**Comment by Starystars67** [comment]
Agreed not 2.4.0 worthy. I suspect that the Lua rework will be 2.4.0 in my opinion.
Otherwise nice work and all approved. Merge when ready @lionkor

**Pull request merged**